### DISCUSSION TITLE
Code for dragon 32167

### DISCUSSION THREAD

**Discussion by Buatdownloadpelm** [ihhub/fheroes2#3824] [General]
Hi,

Thanks for the project so i can play very old game, and when i play this game i remember i can get instantly 5 dragon with this code, but i try using it in the game now can not get the drgon anymore, is there any new code to get the drgaon?

Sorry for my poor english

Thanks.

**Comment by ihhub** [comment]
Hi @Buatdownloadpelm , fheroes2 doesn't support cheat codes as of now.

**Reply by Buatdownloadpelm** [reply]
Ooo ok , thanks for the explanation, are this will be support for next update version?

**Reply by ihhub** [reply]
This might be supported but not in our plans. After all the best way to enjoy the game is to play without cheats, isn't it?😉

**Reply by Zombie-Ryushu** [reply]
Actually I am unable to get myself past the second level because the cheats don't work. I've beaten the original build, but on fheroes 2 I cant get past the second level.

**Reply by ihhub** [reply]
Hi @Zombie-Ryushu , we had such issue and fixed it few days ago. Please read #3814 where you can find even a solution for your problem.
